frontend/src/components/Header.js

```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from 'react';
 import { NavLink } from 'react-router-dom';
 import '../scss/Header.scss';
+import Button from './ui-kit/components/Button/Button';
```

Comment:
Since we have an index.js file, you don't need to go directly to the file, just to the folder